### DISCUSSION TITLE
[Enhancement] Fluent UI support

### DISCUSSION THREAD

**Discussion by antosubash** [dotnet/maui#188] [General]
## Summary
https://developer.microsoft.com/en-us/fluentui#/

maui support for Fluent UI

**Comment by GiampaoloGabba** [comment]
What really bugs me about fluent ui is that Microsoft implemented it for iOS, Android and even REACT NATIVE (a competitor framework) but not for their own multiplatform framework (xamarin "native" and Xamarin Forms).

Sometimes Microsoft feels so.... disconnected... i dont know.

I get that they love so much more react native instead of their own multiplatform framework, but... come on...

**Comment by mrwcjoughin** [comment]
Once again Microsoft did not develop Xamarin or Xamarin.Forms - they
purchased the Xamarin Corporation a few years ago
